src/hooks/useCart/useCart.tsx

```diff
@@ -138,11 +137,6 @@ export const useCart = (
     const fetchQuota = async (): Promise<void> => {
       setCartState("FETCHING_QUOTA");
       try {
-        if (products.length === 0) {
```

Comment:
Hmm I am not sure but might there be a scenario where we hit useCart and products is empty?

Reply:
hmm, currently the QR provides the products, and if not there's default products on the backend, unlikely we will reach useCart without products i feel!

Reply:
yea, given that the retrieval of products is now earlier on, there won't be the case when products is empty. Previously we wait till this part of the flow to retrieve products